Protocol.md

```diff
@@ -605,7 +605,11 @@ send this message to the server. The client MUST set the *your_cookie*
 field to the cookie the server has used in the nonce of the
 'server-hello' message. It SHALL also set the *subprotocols* field to
 the exact same `Array` of subprotocol strings it has provided to the
-WebSocket client implementation for subprotocol negotiation.
+WebSocket client implementation for subprotocol negotiation. If the
+user application requests to be pinged in a specific interval, the
+client SHALL set the field *ping_interval* to the requested ping
+interval in seconds. Otherwise, *ping_interval* MUST be set to `0`
+indicating that no WebSocket *ping* messages SHALL be sent.
```

Comment:
We don't have that for `max_packet_size` (bytes) either and seconds should be clear enough. :)